source/_includes/common-tasks/beta_version.md

```diff
@@ -52,24 +52,24 @@ docker pull {{ site.installation.container }}:beta
 
 1. Stop the Home Assistant service.
 
-2. Switch to the user that is running Home Assistant
+2. Switch to the user that is running Home Assistant.
```

Comment:
> Consider using “who” when referring to a person for grammatical correctness.
> 
> ```diff
> - Switch to the user that is running Home Assistant.
> + Switch to the user who is running Home Assistant.
> ```
> 
> Committable suggestion

Disagree. The user in this context is not an actual person but a conceptual entity representing the user in the software.

Reply:
> @c0ffeeca7 Thank you for your clarification. You are correct in pointing out that in this context, "user" refers to a conceptual entity rather than an actual person. Therefore, using "that" is indeed grammatically appropriate here. I appreciate your attention to detail and your explanation. If there's anything else you'd like to discuss or need further assistance with, feel free to let me know!

Can you remember this for future reviews? Not to replace 'that' by 'who' when referring to the user concept?